bin/Makefile.am

```diff
@@ -60,25 +60,14 @@ regenerate: \
 
 ## Create compatibility symlinks if available
 
-xpmroot_name =`echo "xpmroot" | "$(SED)" -e "${transform}"`
 fvwm_root_name =`echo "fvwm-root" | "$(SED)" -e "${transform}"`
 FvwmCommand_name =`echo "FvwmCommand" | "$(SED)" -e "${transform}"`
 
 install-exec-local:
```

Comment:
If `install-exec-local` is being used just for `xpmroot` root, then this line can go as well.